worldcoin/world-id-docs
# Dev-1093-fix-seo-issue-on-docs

### Context
We identified several broken paths (404) in the documentation based on SEO traffic reports. This PR adds redirects for the following paths to ensure users are redirected to the correct locations

- `/quick-start` -> https://docs.world.org/world-chain/quick-start'
- `/id/incognito-actions` -> https://docs.world.org/world-id/id/cloud
- `/id/idkit` -> 'https://docs.world.org/world-id/reference/idkit'
- `/apps/discord` -> 'https://discord.com/invite/worldnetwork'
- `/api-docs` -> 'https://docs.world.org/world-id/reference/api'
- `/advanced/zero-knowledge-proofs` -> 'https://docs.world.org/world-id/further-reading/zero-knowledge-proofs'
- `/advanced/privacy` -> 'https://vault.pactsafe.io/s/8a18d792-fd76-44db-9b92-b0bb7981c248/legal.html#contract-s1ytru6kk'
- `/advanced/on-chain` ->'https://docs.world.org/world-id/id/on-chain'

### Changes
- Added redirects in next.config.mjs
- Tested redirects locally to confirm they work as expected.

### How to test
- Run the application locally.
- Visit the old paths and verify they correctly redirect to the new locations.
